plugins/source/aws/resources/services/glue/databases.hcl

```diff
@@ -40,6 +41,10 @@ resource "aws" "glue" "databases" {
     description       = "The Amazon Resource Name (ARN) of the workflow."
     generate_resolver = true
   }
+  userDefinedColumn "tags" {
+    type              = "json"
```

Comment:
Do we need a description?

Reply:
Done in https://github.com/cloudquery/cloudquery/pull/1326/commits/27abbf9fa677e939adac71327e40e5dda8c99957!